### PR TITLE
Add download data script

### LINKED ISSUE #1325: Polygon User Guide Long CSV Download Output
Under the "[Realistic Example](https://datashader.org/user_guide/Polygons.html#realistic-example)" in the Polygon User Guide, the CSV download is taking up the majority of the page.

image

It continues like this for about 80% of the entire page.

### PR DESCRIPTION
**Pull request by hoxbro**
Fixes #1325

### CODE REVIEW

**Comment by Hoxbro** [comment]
I have built the documents again from this PR. Which fixes the problem in the issue.

**Comment by hoxbro** [comment]
need to update to work with bokeh_sampledata

File: setup.py [hunk 1]
```diff
@@ -40,6 +40,11 @@
     'spatialpandas',
 ]
 
+if sys.python_version_info[:2] >= (3, 10):
```
Comment (hoxbro) [inline setup.py:43]:
```suggestion
if sys.version_info[:2] >= (3, 10):
```

**Review by philippjfr — APPROVED** [review]

**Comment by philippjfr** [comment]
What's the longer term plan? Should the `pyct` commands in `datashader/__init__.py` be removed?

**Comment by hoxbro** [comment]
> What's the longer term plan?

This only collects all the download data procedures in one place, it is heavily focused on the developer experience. `pyct` is still used.

I think longer term `pyct` should be replaced by `hvsampledata`.

**Pull request merged**